### DISCUSSION TITLE
Inventory hovered item highlight / grid enhancements

### DISCUSSION THREAD

**Discussion by AnthonyPoschen** [runelite/runelite#16555] [Ideas]
when mousing over items in the inventory dead space exists between the items.

For many players trying to get good at swapping gear it may be hard to know when you are actually hovering the mouse over the item as some items icon's have space on some sides but are pixel perfect with the border on the other side. 

Having the ability to have the inventory slot highlight in a customizable way in terms of colour and opacity as well as allowing the highlight to be behind the items as a backdrop would be fantastic to help players visually see when they can click on an item without mouse tooltips on.

This would be an enahcnement to the inventory grid plugin.

Being able to configure the same settings for both dragging and simply hover would be great.

**Comment by ElijahMe** [comment]
Hello! This plugin would be great! I would like to add possibly, highlight when you hover over anything in your inventory, ex. Gear, Tabs, Especially Prayers, and more! It could be a glow trim ore just a colored square. and light up on successful click etc.

**Comment by Mac10Monarch** [comment]
A simple extension of the plugin to give users the option to highlight the tile when a player hovers over an inventory cell would be nice. Furthermore, there could also be a usage behind keeping the entire grid highlighted at all times.

**Comment by Drerrday** [comment]
is this made yet? I see this is a discussion from 3 years ago
